File: lib/cutting_lib/src/main/java/com/cutting/machine/communication/KeyInfoBase.java
package com.cutting.machine.communication;

import com.cutting.machine.bean.KeyAlign;
import com.cutting.machine.bean.KeyType;

/* loaded from: classes2.dex */
public interface KeyInfoBase {
    int getExtDoublekeyDepthSteps();

    KeyAlign getKeyAlign();

    KeyType getKeyType();

    int getKeyWidthSteps();

    int getLengthSteps();

    int getShoulderBlock();

    int getSide();

    int getThickSteps();

    boolean isNewHonda();
}

File: app/src/main/java/com/cutting/machine/communication/KeyInfoBase.kt
package com.cutting.machine.communication

import com.cutting.machine.bean.KeyAlign
import com.cutting.machine.bean.KeyType

/* loaded from: classes2.dex */
interface KeyInfoBase {
    val extDoublekeyDepthSteps: Int

    val keyAlign: KeyAlign?

    val keyType: KeyType?

    val keyWidthSteps: Int

    val lengthSteps: Int

    val shoulderBlock: Int

    val side: Int

    val thickSteps: Int

    val isNewHonda: Boolean
}
